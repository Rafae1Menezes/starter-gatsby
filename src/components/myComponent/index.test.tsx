import React from "react";
import renderer from "react-test-renderer";
import { MyComponent } from ".";

describe("MyComponent", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<MyComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
