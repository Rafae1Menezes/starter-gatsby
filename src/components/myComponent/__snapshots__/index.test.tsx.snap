// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent renders correctly 1`] = `
[
  <h2
    className="titulo"
  >
    Meu Componente
  </h2>,
  <div
    className="images"
  >
    <img
      alt="Artistas"
      height={346}
      placeholder="blurred"
      src="../../images/artistas.png"
      width={374}
    />
    <test-file-stub
      className="image"
    />
  </div>,
]
`;
